stack_test.go

```diff
@@ -518,6 +526,33 @@ func BenchmarkTrace100(b *testing.B) {
 	}
 }
 
+func BenchmarkCallers(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		stack.Callers(1)
+	}
+}
+
+func BenchmarkCallers10(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		b.StopTimer()
+		deepCallers(10, b)
+	}
+}
+
+func BenchmarkCallers50(b *testing.B) {
+	b.StopTimer()
```

Comment:
We should call `b.StopTimer` inside the loop before calling `deepCallers` like it is in `BenchmarkCallers10`. I realize you followed the pattern I had in the BenchmarkTrace functions but I did it wrong there and never noticed until now. Please fix those too if you don't mind.